generative_ai/embeddings/document_retrieval_example.py

```diff
@@ -19,27 +19,41 @@
 
 
 def embed_text() -> list[list[float]]:
-    """Embeds texts with a pre-trained, foundational model.
+    """Calculates embeddings for the given text inputs.
 
     Returns:
-        A list of lists containing the embedding vectors for each input text
+        A list of lists containing the embedding vectors for each input text.
     """
 
-    # A list of texts to be embedded.
-    texts = ["banana muffins? ", "banana bread? banana muffins?"]
```

Comment:
The motivation to shift to a longer, more complex texts is the following:

* The new texts are more representative of real-world documentation or help articles that might be used in a text embedding scenario. This should serve as a subtle hint at potential applications of Vertex AI in improving search and retrieval for various help documents.
* The longer, more detailed texts provide a better demonstration of how the embedding model can handle complex, multi-sentence inputs.
* The new text examples specifically mention Google products and services, to subtly showcase Google's ecosystem.